docs/maui/views/MediaElement.md

```diff
@@ -152,6 +183,30 @@ By default, the media that is defined by the `Source` property doesn't immediate
 
 Platform provided media playback controls are enabled by default, and can be disabled by setting the `ShouldShowPlaybackControls` property to `false`.
 
+### Using MetaData
+
+A `MediaElement` can use metadata for `MediaElement.MetaDataTitle`, `MediaElement.MetaDataArtist` and `MediaElement.MetaDataArtworkUrl`. You can set
+the title or artist to show what is currently playing on lockscreen controls for Windows, Mac Catalyst, iOS, and Android. You can set a local or 
+remote URL with artwork for the lockscreen. It should be at least 1080P for best quality to be displayed. It must be a URL and be either `.jpg` or
+ `.png` 
+
+```xaml
```

Comment:
I believe this has to be `xml`. I don't think the syntax highlighting engine knows `xaml`.

I could be wrong, though! Shaun + Gerald have more expertise with the docs than me.

```suggestion
```xml
```

Reply:
Please look at Line:229 where there is an example of xaml and Line:288 for another. If this is wrong I'm not sure
what we should do?